runtime/compiler/env/VMJ9.cpp

```diff
@@ -4492,7 +4492,7 @@ TR_J9VMBase::initializeLocalArrayHeader(TR::Compilation * comp, TR::Node * alloc
    else if (!comp->getOptions()->realTimeGC() && instanceSize == 0)
       {
       // Contiguous size field is zero (mandatory)
-      // For J9VM_GC_COMBINATION_SPEC only 0 size discontinous arrays are supported
+      // For J9VM_GC_COMBINATION_SPEC only 0 size discontinuous arrays are supported
```

Comment:
The correct term should be "discontiguous".  It was wrong in the first place.

Reply:
fwiw, those are from: e8cc1e8, ae808d4, and 9c69e1b.
Thanks for the correction.